### PR TITLE
javascript-javascript2-week2

### PR DESCRIPTION
**Pull request by Hailegebriel22**

### CODE REVIEW

File: javascript/javascript2/week2/moviehomework.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
+let newNumbers = numbers.filter(element => element % 2 != 0)
```
Comment (shravanipagilla) [inline javascript/javascript2/week2/moviehomework.js:2]: seems you forgot to define numbers array:
let numbers = [1, 2, 3, 4];  I added this and got right output.
very nice you used arrow function with filter and map .

File: javascript/javascript2/week2/moviehomework.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
...ne of the Guys", "year": 1985, "rating": 6.5, "votes": 10976, "running_times": 6000 }, { "title": "Just Visiting", "year": 2001, "rating": 5.8, "votes": 16255, "running_times": 5280 }, { "title": "Just Wright", "year": 2010, "rating": 5.9, "votes": 11239, "running_times": 6000 }, { "title": "Justice League", "year": 2017, "rating": 7.2, "votes": 141370, "running_times": 7200 }, { "title": "Justin Bieber: Never Say Never", "year": 2011, "rating": 1.6, "votes": 74720, "running_times": 6900 }, { "title": "K-19: The Widowmaker", "year": 2002, "rating": 6.7, "votes": 52505, "running_times": 8280 }, { "title": "K-9", "year": 1989, "rating": 6, "votes": 27515, "running_times": 6060 }, { "title": "K-PAX", "year": 2001, "rating": 7.4, "votes": 160033, "running_times": 7200 }, { "title": "Kabhi Alvida Naa Kehna", "year": 2006, "rating": 6.1, "votes": 15518, "running_times": 11580 }, { "title": "Kabhi Haan Kabhi Naa", "year": 1994, "rating": 8, "votes": 14320, "running_times": 9480 }, { "title": "Kabhi Khushi Kabhie Gham...", "year": 2001, "rating": 7.5, "votes": 35341, "running_times": 12600 }, { "title": "Kaboom", "year": 2010, "rating": 5.7, "votes": 11187, "running_times": 5160 }, { "titl
```
Comment (shravanipagilla) [inline javascript/javascript2/week2/moviehomework.js]: seems you missing log out 'averageRating'

Comment (shravanipagilla) [inline javascript/javascript2/week2/moviehomework.js]: The expected output will be like : {goodMovies: 33, averageMovies: 45, goodMovies: 123}
please check your code again for requirement 9.

**Review by shravanipagilla — CHANGES_REQUESTED** [review]

File: javascript/javascript2/week2/hyfindex.html [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+      <img class="logo" src="https://thumbs.dreamstime.com/z/new-product-item-store-25048588.jpg" alt="Yellow new product bage">
+      <a href="#">Home</a>
+      <a href="#">About</a>
+      <a href="#">Contact</a>
+    </div>
+  </header>
+  <main> 
+    <div id="search-wrapper"> 
+      <input type="search" id="search-bar"  placeholder="Search for a product">
+      <input type="number" id="max-price"  placeholder="Set Maximum price">
```
Comment (shravanipagilla) [inline javascript/javascript2/week2/hyfindex.html:22]: code looks good.
however : These two number fields allowing user to enter -values. you might restrict user entering minus numbers.

**Review by shravanipagilla — CHANGES_REQUESTED** [review]

File: javascript/javascript2/week2/moviehomework.js (hunk 2 above)
Comment (Hailegebriel22) [reply javascript/javascript2/week2/moviehomework.js]: I don't understand. How come 6527 movies tagged  based on the rating: Good (>= 7), Average (>= 4 //and < 7), Bad (< 4) and  expected output will be only  goodMovies: 33, averageMovies: 45, goodMovies: 123. 
shouldn't the sum of the tags equal the length of movies array?
shouldn't be something like this {Good: 2602, Average: 3837, Bad: 88}

**Pull request merged**